HW3/PismakArchip/hw2.py

```diff
@@ -0,0 +1,14 @@
+number = int(input("Введіть чотиризначне натуральне число: "))
+
+digits = [int(digit) for digit in str(number)]
+
+product_of_digits = digits[0] * digits[1] * digits[2] * digits[3]
+
+reversed_number = int(str(number)[::-1])
+
+sorted_digits = sorted(digits)
+sorted_number = int("".join(map(str, sorted_digits)))
+
+print(f"Добуток цифр: {product_of_digits}")
+print(f"Число у зворотному порядку: {reversed_number}")
+print(f"Цифри, відсортовані у зростаючому порядку: {sorted_number}")
```

Comment:
look at this implementation
number = int(input("Enter four digit number: "))
num_1 = number // 1000
num_2 = (number // 100) % 10
num_3 = (number // 10) % 10
num_4 = number % 10
product = num_1 * num_2 * num_3 * num_4
print(product)